src/components/ForgotPassword/ForgotPassword.styles.js

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import Form from 'components/Form';
+
+export const Input = styled(Form.Input)`
+  & > input {
+    width: 100%;
+  }
+`;
+
+export const Legend = styled.p`
+  color: ${(props) => props.theme.color.gray700};
```

Comment:
```suggestion
  color: ${({ theme }) => theme.color.gray700};
```
In my opinion, destructuring theme looks better in these cases. What do you think?